AgentInterfaces/SessionHostsStartInfo.cs

```diff
@@ -59,6 +62,11 @@ public class SessionHostsStartInfo
         /// </summary>
         public string FQDN { get; set; }
 
+        /// <summary>
+        /// List of Public IP Addresses
+        /// </summary>
+        public List<PublicIpAddress> PublicIpAddresses { get; set; }
```

Comment:
this will contain the one in the PublicIpV4Address, right?

Reply:
(just curious)

Reply:
Correct. I will contain all of them. If there is only one, it will use the PublicIpV4Address one